frontend/src/app/component/phenotypic-feature/age-control/date/date.component.html

```diff
@@ -0,0 +1,31 @@
+<table class="full-width" cellspacing="0">
+    <tr>
+      <td>
+        <mat-form-field class="full-width" appearance="fill">
+          <mat-label>Years</mat-label>
+          <input matInput [formControl]="yearFormControl" type="number" placeholder="Ex. 1990" min="1" [errorStateMatcher]="matcher">
+          <mat-error *ngIf="yearFormControl.hasError('year') && !yearFormControl.hasError('required')">
+            Please enter a valid year
+          </mat-error>
+        </mat-form-field>
+      </td>
+      <td>
+        <mat-form-field class="full-width" appearance="fill">
+          <mat-label>Months</mat-label>
+          <input matInput [formControl]="monthFormControl" type="number" placeholder="Ex. 1" min="1" max="12" [errorStateMatcher]="matcher">
+          <mat-error *ngIf="monthFormControl.hasError('month') && !monthFormControl.hasError('required')">
+            Please enter a valid month
+          </mat-error>
+        </mat-form-field>
+      </td>
+      <td>
+        <mat-form-field class="full-width" appearance="fill">
+          <mat-label>Days</mat-label>
+          <input matInput [formControl]="dayFormControl" type="number" placeholder="Ex. 27" min="1" max="31" [errorStateMatcher]="matcher">
```

Comment:
Minimum and the default value is `0`, to cover a case of e.g. 5 years and 3 months.